tests/Middleware/SanitizeHttpHeadersMiddlewareTest.php

```diff
@@ -25,10 +26,17 @@ public function testInvoke($inputData, $expectedData)
     {
         $event = new Event(new Configuration());
         $event->setRequest($inputData);
+        $request = $this->prophesize(ServerRequestInterface::class)->reveal();
```

Comment:
We don't use `prophecy` in the codebase and I would like to stay away from it as its usage is not as clear as the traditional mock builder (e.g. the code completion does not work) and it could be a stop blocker for new contributors

Reply:
I personally prefer Prophecy, and for autocompletion I personally use a PHPStorm plugin. But consistency is better, I'll change this.